INSTALL.md

```diff
@@ -103,3 +111,24 @@ make py-deps
 ```
 
 See [GitHub Actions workflow](.github/workflows/main.yml) for other usages of the Makefile.
+
+## Tests
+
+To test the WebAssembly module we will use the [cypress](https://www.cypress.io/) test framework.
```

Comment:
Can you add a sentence explaining why we want end to end testing?

Reply:
Done in 89694883e32a2e3f06cf6a183969600d3e952131